src/utils/ConfigManager.ts

```diff
@@ -0,0 +1,58 @@
+import path, {dirname} from "node:path";
+import {fileURLToPath} from "node:url";
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export interface UI5LintConfigType {
+	ignores?: string[];
+};
+
+const CONFIG_FILENAMES = [
+	"ui5lint.config.js",
+	"ui5lint.config.mjs",
+	"ui5lint.config.cjs",
+];
+
+export default class ConfigManager {
+	#projectRootDir: string;
+	#configFile: string | null;
+
+	constructor(projectRootDir: string, configFile?: string) {
+		if (!projectRootDir) {
+			throw Error("Project's root dir is required");
+		}
+		this.#projectRootDir = projectRootDir;
+		this.#configFile = configFile ?? null;
+	}
+
+	#resolveModulePaths(fileName: string): string {
+		// Node on Windows behaves strange, does not work with absolute paths
+		// and modifies files extensions in tests i.e. js -> ts, mjs -> tjs
+		// Keeping the relative path in POSIX format resolves those issues.
+		return path.posix.join(
+			path.relative(__dirname, this.#projectRootDir).replaceAll(path.win32.sep, path.posix.sep),
+			fileName);
+	}
+
+	async getConfiguration(): Promise<UI5LintConfigType> {
+		let config = {} as UI5LintConfigType;
+
+		if (this.#configFile) {
+			// Relative paths are needed to make it work on Windows
+			const configFilePath = this.#resolveModulePaths(this.#configFile);
+			({default: config} = await import(configFilePath) as {default: UI5LintConfigType});
+		} else {
+			// Find configuration file
+			({default: config} = await Promise.any(
+				CONFIG_FILENAMES.map(
```

Comment:
Not sure whether this works, but can't we just import `ui5lint.config` and let Node.ja decide on the extension?

Reply:
The issue is that if we don't provide file extension, it cannot resolve the configurations with `mjs` and `cjs` extensions.